packages/noports_core/lib/src/common/mixins/at_client_bindings.dart

```diff
@@ -3,19 +3,26 @@ import 'package:at_utils/at_utils.dart';
 
 mixin AtClientBindings {
   AtClient get atClient;
+
   AtSignLogger get logger;
 
   Future<void> notify(
     AtKey atKey,
-    String value,
-  ) async {
-    await atClient.notificationService
-        .notify(NotificationParams.forUpdate(atKey, value: value),
-            onSuccess: (NotificationResult notification) {
-      logger.info('SUCCESS:$notification with key: ${atKey.toString()}');
-    }, onError: (notification) {
-      logger.info('ERROR:$notification');
-    });
+    String value, {
+    required bool checkForFinalDeliveryStatus,
+    required bool waitForFinalDeliveryStatus,
+  }) async {
```

Comment:
I added these params as required and dealt with the ripple effects as it is better in general to have these parameters explicitly exposed